public/pages/Aliases/containers/Aliases/Aliases.tsx

```diff
@@ -145,6 +152,8 @@ class Aliases extends Component<AliasesProps, AliasesState> {
       aliasCreateFlyoutVisible: false,
       aliasEditFlyoutVisible: false,
       editingItem: null,
+      dataSourceId: this.props.dataSourceId,
+      dataSourceLabel: this.props.dataSourceLabel,
```

Comment:
I think using the props is good enough, there is no place to `set` these two states.

Reply:
Got it, I agree we do not need to store in the state and props are sufficient. However, aliases/indices/templates/component Templates/data streams update the URL by comparing the previous and the current state and also update the tables with the latest data. The state is also used to update the URL as well. I expanded the state with dataSourceId & dataSourceLabel, to support the current MDS with minimal modifications to the code. 

If we use props, we need to compare the previous props & current props and have to update the URL with the new dataSourceId & dataSourceLabel with additional code and also during component mounting we have to do the same.